src/web/page/mod.rs

```diff
@@ -268,3 +263,47 @@ mod tests {
         assert!(Url::parse(&safe).is_ok());
     }
 }
+
+// --- Tera ---
+
+mod templates;
+
+pub(crate) use templates::TemplateData;
+
+lazy_static::lazy_static! {
+    /// Holds all data relevant to templating
+    pub(crate) static ref TEMPLATE_DATA: TemplateData = TemplateData::new().expect("Failed to load template data");
+}
```

Comment:
Can we move all of this at the top of the file, and merge it with the existing imports/mod declarations/statics? It's weird having to find them in the middle of the file.

Reply:
Well, it's easier for me personally to keep them separate like this (hence the comment above them all) since that'll be all that's left in the file once I'm done. Everything above the comment will be removed since `Page` and all the stuff that goes with it gets replaced